index.js

```diff
@@ -59,6 +60,11 @@ module.exports = function(source) {
     }
   });
 
+  manualImports.forEach(function(dep) {
+    var fullPath = utils.lookup(dep, options.paths, '', true);
+    self.addDependency(fullPath);
```

Comment:
we need only `manualImports = manualImports.map(function (it) { return utils.lookup(it, options.paths, '', true); });` before, or something to create full paths to those imports